test/subscriptions.jl

```diff
@@ -13,10 +12,10 @@ end
 
 function do_nothing_localhost()
     @async HTTP.listen(HTTP.Sockets.localhost, 8081) do http
-        if HTTP.WebSockets.is_upgrade(http.message)
+        if HTTP.WebSockets.isupgrade(http.message)
             HTTP.WebSockets.upgrade(http) do ws
-                while !eof(ws)
-                    data = readavailable(ws)
+                for data in ws
+                    nothing;
```

Comment:
Does the semi colon add anything here?

Reply:
not really, it just looked nicer at the time 😅